src/Microsoft.ML.Data/Transforms/ColumnConcatenatingEstimator.cs

```diff
@@ -10,9 +10,28 @@
 
 namespace Microsoft.ML.Transforms
 {
+
     /// <summary>
-    /// Concatenates columns in an <see cref="IDataView"/> into one single column. Estimator for the <see cref="ColumnConcatenatingTransformer"/>.
+    /// Concatenates one or more input columns into a new output column.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Any, except [key type](xref:Microsoft.ML.Data.KeyDataViewType). All input columns must have the same type.  |
+    /// | Output column data type | A vector of the input columns' data type |
+    ///
+    /// The resulting [ColumnConcatenatingTransformer](xref:Microsoft.ML.Data.ColumnConcatenatingTransformer) creates a new column,
+    /// named as specified in the output column name parameters, where the input values are concatenated in a vector.
```

Comment:
>vector [](start = 110, length = 6)

I would drop few words regarding concatenation order.
We perform it in order of column names, which is somewhat obvious, but still probably worth to mention.

Reply:
Good point!

---
In reply to: [277077542](https://github.com/dotnet/machinelearning/pull/3428#discussion_r277077542) [](ancestors = 277077542)